docs/migration/v4.md

```diff
@@ -36,4 +36,12 @@ In `v3` the `IRuntimeRegistry` offered the ability to use the whole `NCronJobOpt
 runtimeRegistry.AddJob(n => n.AddJob<MyJob>(...).RunAtStartup().AddExceptionHandler<MyHandler>());
 ```
 
-It didn't make sense to add a startup job during runtime. Also adding exception handlers during runtime was out of scope for this feature. Therefore the interface is more restrictive now and only allows to add jobs.
```

Comment:
Typo above:

"In `v3` the `IRuntimeRegistry` offered the ability to use the whole `NCronJobOptionBuilder` which led to confusion, especially if done something like this:"